### ISSUE TITLE
Cumulative Return by Quantile for each day/period

### ISSUE THREAD

**Issue by luca-s** [quantopian/alphalens#66]
Cumulative Return by Quantile plot

Only the 1 day forward returns graph is plotted. I expected to see plots for each day/period forward return ('days' argument passed to create_factor_tear_sheet ) .
I might have a factor that shows alpha only after X or Z days and I might not be interested to see 1 day forward returns plots, so I would pass days = (X, Z) to create_factor_tear_sheet and I would expect to see the plots for X and Z forward returns. At least it would be nice to have this feature as an option if not the default behaviour.

**Comment by luca-s** [comment]
I understand that is not straightforward to decide the best way to plot the cumulative daily returns plot when the forward returns have periods greater than 1 day (as the forward returns of subsequent days overlaps), but I noticed the code saves the forward returns for any period length as **daily** returns (please see #74) so the returns are already in a format suitable to be passed to the  plotting function (plotting.plot_cumulative_returns_by_quantile). Plotting the daily returns should be a reasonable compromise.

The other, more complex, option would be to calculate the daily returns as the mean of all the overlapping returns that influence that day: for a period of N days each daily returns  would be calculated as the average of N forward returns (day 1 returns of '1 day ago' factor, day 2 returns of '2 day ago' factor, day 3 returns of '3 day ago' factor and so on until N day ago factor). But that wouldn't give us much more information. 

This also applies to  #65.

**Comment by luca-s** [comment]
Actually It makes me laugh that I defined "more complex" the second option. it simply requires to call rolling_mean on the daily return dataframe.

**Comment by jameschristopher** [comment]
This and #65 I think are great. It is definitely misleading to look at the cumulative returns graph and see it looking terrible even though the signal was defined for a monthly or weekly predictive timeframe. Definitely PR this if you want.

**Comment by luca-s** [comment]
Great, I will provide a PR then. I just hope nobody gets frustrated to see too many plots. We will see.

**Comment by jameschristopher** [comment]
https://github.com/quantopian/alphalens/pull/83
